bookx/settings.py

```diff
@@ -137,7 +137,8 @@
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 
 AUTH_USER_MODEL = 'users.CustomUser'
-
+AUTHENTICATION_BACKENDS = ['users.backends.EmailBackend','django.contrib.auth.backends.ModelBackend']
```

Comment:
Do you still want to keep the `ModelBackend` as well?

Reply:
Just realised I hadn't answered this yet. Mmh I don't really know :) Maybe just so I know where the original one is..